ngx-fudis/projects/ngx-fudis/src/lib/components/link/link.component.html

```diff
@@ -8,12 +8,16 @@
 
 <ng-template #anchorLink>
 	<ng-container *ngIf="!routerLinkUrl">
-		<a [href]="href" class="fudis-link__anchor fudis-link__anchor__{{ size }} fudis-link__anchor__{{ color }}">
+		<a
+			[class.fudis-link__anchor--disabled]="disabled"
```

Comment:
Screen reader should now also be notified that the link is disabled, use `attr.aria-disabled`.
And same addition to all three link variations.